### PR TITLE
[16.0][MIG] sale_line_returned_qty: Migration to 16.0

### PR DESCRIPTION
**Pull request by pausanchezqubiq**

### CODE REVIEW

**Review by ferran-S73 — APPROVED** [review]

File: sale_line_returned_qty/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+This module adds the returned quantity of sale order lines as a computed field.
```
Comment (rousseldenis) [inline sale_line_returned_qty/readme/DESCRIPTION.rst]: More than a technical explanation, this should be more a functional one (what it does, where it is displayed).

Comment (pausanchezqubiq) [reply sale_line_returned_qty/readme/DESCRIPTION.rst]: Hey @rousseldenis, the description is updated.

**Comment by JSebastian7913** [comment]
Functional review done.

**Review by rousseldenis — APPROVED** [review]
Code review

**Review by chusamo — APPROVED** [review]
LGTM

**Comment by pausanchezqubiq** [comment]
I uploaded it again at https://github.com/OCA/stock-logistics-workflow/pull/1309 as the previous commits were not all there.

**Pull request closed without merging**